_codux/boards/card-collection.board.css

```diff
@@ -0,0 +1,100 @@
+.cardCollection {
+    display: flex;
+    padding: 48px;
+    flex-direction: column;
+    align-items: center;
+    font-family: Arial, sans-serif;
+}
+
+.cardCollectionHeader {
+    font-size: 70px;
+    text-align: center;
+    font-weight: 700;
+    text-wrap: pretty;
+
+    @media (width<= 768px) {
+        font-size: 60px;
+    }
+    @media (width<= 480px) {
+        font-size: 30px;
+    }
+}
+
+.cardCollectionItems {
+    display: flex;
+    margin-top: 4vh;
+    gap: 1.5rem;
+    flex-direction: row;
+
+    @media (width<= 768px) {
+        flex-direction: column;
+    }
+}
+
+.cardCollectionCard {
+    flex-wrap: wrap;
+    max-width: 100%;
+    border-radius: 8px;
+    overflow: hidden;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    flex-direction: column;
+    cursor: pointer;
+    transition: box-shadow 0.3s;
+
+    @media (width<= 480px) {
+        font-size: 30px;
+    }
+}
+
+.cardCollectionImage {
+    object-fit: cover;
+}
+
+.cardCollectionContent {
```

Comment:
you could avoid adding this extra container and class by using utilizing flex and gap in `cardCollectionCard`